flink-core/src/main/java/org/apache/flink/configuration/NettyShuffleEnvironmentOptions.java

```diff
@@ -175,7 +175,8 @@
 
 	public static final ConfigOption<Integer> NUM_ARENAS =
 		key("taskmanager.network.netty.num-arenas")
-			.defaultValue(-1)
+			.intType()
+			.defaultValue(1)
```

Comment:
It seems that you changed the default value without updating/rebuilding the docs.
These's a test case (`ConfigOptionsDocsCompletenessITCase`) failed because of this.